integrations/extensions/starter-kits/servicenow/openapi.sn.json

```
@@ -0,0 +1,684 @@
+{
+  "openapi": "3.0.3",
+  "info": {
+    "title": "ServiceNow Blog",
+    "version": "1.0.0",
+    "contact": {}
+  },
+  "servers": [
+    {
+      "url": "https://{instance}.service-now.com",
+      "variables": {
+        "instance": {
+          "default": "dev93773",
+          "description": "Your PDI name"
+        }
+      }
+    }
+  ],
+  "components": {
+    "securitySchemes": {
+      "basicAuth": {
+        "type": "http",
+        "scheme": "basic"
+      }
+    }
+  },
+  "security": [
+    {
+      "basicAuth": []
+    }
+  ],
+  "paths": {
+    "/api/now/table/incident": {
+      "post": {
+        "summary": "https://dev93773.service-now.com/api/now/table/incident",
+        "description": "https://dev93773.service-now.com/api/now/table/incident",
+        "operationId": "httpsdev93773ServiceNowComApiNowTableIncident",
+        "requestBody": {
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "assignment_group": {
+                    "type": "string",
+                    "example": "287ebd7da9fe198100f92cc8d1d2154e"
+                  },
+                  "impact": {
+                    "type": "string",
+                    "example": "2"
+                  },
+                  "short_description": {
+                    "type": "string",
+                    "example": "Unable to connect to office wifi"
+                  },
+                  "urgency": {
+                    "type": "string",
+                    "example": "2"
+                  }
+                }
+              },
+              "example": {
+                "assignment_group": "287ebd7da9fe198100f92cc8d1d2154e",
+                "impact": "2",
+                "short_description": "Unable to connect to office wifi",
+                "urgency": "2"
+              }
+            }
+          }
+        },
+        "responses": {
+          "201": {
+            "description": "https://dev93773.service-now.com/api/now/table/incident",
```

Prob should replace these ones too
```suggestion
            "description": "Successful operation",
```

---

I deliberately left that since this is how the convert tool works

---

See https://github.com/watson-developer-cloud/assistant-toolkit/pull/20#discussion_r860263359